src/sentry/tasks/post_process.py

```diff
@@ -562,6 +562,7 @@ def get_event_raise_exception() -> Event:
         sentry_sdk.set_tag("is_reprocessed", is_reprocessed)
 
         is_transaction_event = event.get_event_type() == "transaction"
+        is_error_event = event.get_event_type() == "error"
```

Comment:
Yep, that looks right to me https://github.com/getsentry/sentry/blob/master/src/sentry/eventtypes/error.py#L27

Reply:
Sorry, where are you getting `default`?

Reply:
Is it because of this line https://github.com/getsentry/sentry/blob/master/src/sentry/eventstore/models.py#L231?

Reply:
Got it. In case we haven't already -- let's double check that error events indeed have the `exception` key? So that we know for sure `error` is the right check here

Reply:
If we are unsure, one option would be to exclude transactions events (check the existing `is_transaction_event` variable) rather than including only errors.

That said, I don't know the code well enough to give a recommendation and I trust you folks to make the right call.

Reply:
yeah, i would agree, we should simply look to see if its a transaction event, if that is the intention of the PR.

Reply:
Sounds good, let's do that then @NisanthanNanthakumar if that sounds good to you as well?